### PR TITLE
LPS-134605 add test to test change status on CheckFileEntryMessageListener

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/messaging/test/CheckFileEntryMessageListenerTest.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup();
+
+		_originalName = PrincipalThreadLocal.getName();
+
+		_user = TestPropsValues.getUser();
+
+		PrincipalThreadLocal.setName(_user.getUserId());
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/messaging/test/CheckFileEntryMessageListenerTest.java:75]: See UserTestUtil.setUser(); you should use that to set up the permission checker if possible.

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/messaging/test/CheckFileEntryMessageListenerTest.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-web-test/src/testIntegration/java/com/liferay/document/library/web/internal/messaging/test/CheckFileEntryMessageListenerTest.java]: thanks!

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**